tests/baselines/reference/jsdocImportTypeReferenceToClassAlias.types

```diff
@@ -19,11 +19,11 @@ module.exports.C = C
 /** @param {X} c */
 function demo(c) {
 >demo : (c: X) => void
->c : C
+>c : typeof C
 
     c.s
->c.s : () => void
->c : C
->s : () => void
+>c.s : any

```

Comment:
this seems wrong, which means that this fix may also need to be restricted to types that come through `require`